test/test_parse.py

```diff
@@ -697,13 +831,15 @@ def test_numbers(self):
         self.assertEqual(normalize("that's eighteen nineteen twenty"),
                          "that is 18 19 20")
         self.assertEqual(normalize("that's one nineteen twenty two"),
-                         "that is 1 19 20 2")
+                         "that is 1 19 22")
         self.assertEqual(normalize("that's one hundred"),
-                         "that is 1 hundred")
+                         "that is 100")
         self.assertEqual(normalize("that's one two twenty two"),
-                         "that is 1 2 20 2")
+                         "that is 1 2 22")
         self.assertEqual(normalize("that's one and a half"),
                          "that is 1 and half")
+
+        # TODO fix me
```

Comment:
this test is failing and i could not figure out why yet, this is the main blocker for merging this PR currently, marking as draft until solved